classes/class_tween.rst

```diff
@@ -187,13 +187,22 @@ Tweens are useful for animations requiring a numerical property to be interpolat
 
 Here is a brief usage example that causes a 2D node to move smoothly between two positions:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     var tween = get_node("Tween")
     tween.interpolate_property($Node2D, "position",
             Vector2(0, 0), Vector2(100, 100), 1,
             Tween.TRANS_LINEAR, Tween.EASE_IN_OUT)
     tween.start()
+    
+ .. code-tab:: csharp
+ 
+    var tween = GetNode("Tween") as Tween;
```

Comment:
Don't use `as` if you're not going to do a null check. Prefer a cast `(Tween)GetNode("Tween")` or the equivalent `GetNode<Tween>("Tween")`. The reason for this is that a possible `InvalidCastException` in the place of the cast is much better than a `NullReferenceException` somewhere else.